### ISSUE TITLE
Mod like overwolf.gg

### ISSUE THREAD

**Issue by hmz22** [Aldrin-John-Olaer-Manalansan/DOTA-2-MOD-Master#49]
Can make mode like overwolf.gg inventory changer. 
In overwolf mod can change all item in game and can shuffle for ward or courior or others.
In overwolf i see this files
![16538991394321241038727454958000](https://user-images.githubusercontent.com/36853974/170950583-b85edda6-418b-436d-b44e-a24b0dbd3681.jpg)

**Comment by quishyy** [comment]
Overwolf doesnt "mod" game itself, just injects in dota and you have skins. No one knows how its done, cause it is a secret of developers.

**Comment by hmz22** [comment]
:(

**Comment by ducbh62git** [comment]
> Overwolf doesnt "mod" game itself, just injects in dota and you have skins. No one knows how its done, cause it is a secret of developers. 

This makes the mod more dangerous. It may contains malware or virus.

**Comment by hmz22** [comment]
Can add shuffle mod for global item like ward or courier and show image of item or unlock dota plus?

**Comment by quishyy** [comment]
> > Overwolf doesnt "mod" game itself, just injects in dota and you have skins. No one knows how its done, cause it is a secret of developers. 
> 
> This makes the mod more dangerous. It may contains malware or virus.

Many people report about lags, so idk

**Comment by Aldrin-John-Olaer-Manalansan** [comment]
Maybe that tool automatically generates a MOD(shuffled) in a **Periodic/Scheduled Manner** so that the MOD will always be newer upon next launch.

To answer your question, there are many people whom had suggested the same feature you're suggesting. So let me pin point issues any users will face when this gets implemented(assuming it were implemented here):
1. Everytime we Generate a MOD here at DOTA2 MOD Master, it takes almost 5-15 minutes for the whole operation to finish. During this process, DOTA2 MOD Master Consumes RAM which lags some users.
2. If we've given much thought how **Periodic/Scheduled** Auto Modding works, now imagine the whole MOD Generation operation is always executed periodically/On Schedule. Users will experiece periodic lags during this process.
3. Manual MOD Generations is better because we only Sacrifice RAM when we want to change something.


In conclusion, I might add this feature in the future. But isn't advisable to be used since it could couse periodic lag.

**Comment by hmz22** [comment]
you're right. 
But can add this mod optinal beacuse any player have good hardware can use it. 
New hardaware like ssd and 16 gb ram and so on. 
This good idea for this

**Comment by hmz22** [comment]
For dota plus can unlock with current mod?

**Comment by quishyy** [comment]
> Maybe that tool automatically generates a MOD(shuffled) in a **Periodic/Scheduled Manner** so that the MOD will always be newer upon next launch.
> 
> To answer your question, there are many people whom had suggested the same feature you're suggesting. So let me pin point issues any users will face when this gets implemented(assuming it were implemented here):
> 1. Everytime we Generate a MOD here at DOTA2 MOD Master, it takes almost 5-15 minutes for the whole operation to finish. During this process, DOTA2 MOD Master Consumes RAM which lags some users.
> 2. If we've given much thought how **Periodic/Scheduled** Auto Modding works, now imagine the whole MOD Generation operation is always executed periodically/On Schedule. Users will experiece periodic lags during this process.
> 3. Manual MOD Generations is better because we only Sacrifice RAM when we want to change something.
> 
> 
> In conclusion, I might add this feature in the future. But isn't advisable to be used since it could couse periodic lag.

the problem is how can you add all cosmetics without replacing default ones

**Comment by hmz22** [comment]
Can't find any way without replace deafult?
